s/nub-keyboard/utils/actions.ts

```diff
@@ -0,0 +1,8 @@
+export const actions:any = {
```

Comment:
hmm, i think maybe every application that uses nubs might have its own actions.

i think actions actually need to be passed into a nub-context.